network.go

```diff
@@ -10,3 +10,45 @@ func parseHeaders(headers []interface{}) map[string]string {
 	}
 	return out
 }
+
+type rawHeaders struct {
+	headersArray map[string]string
+	headersMap   map[string][]string
+}
+
+func (r *rawHeaders) Get(name string) string {
+	values := r.GetAll(name)
+	sep := ", "
+	if strings.ToLower(name) == "set-cookie" {
+		sep = "; "
```

Comment:
```suggestion
		sep = "\n"
```
at least we have that upstream.